build.gradle

```diff
@@ -171,9 +169,7 @@ project(":service") {
  */
 static List<String> getServiceProperties(Project project) {
     def commonProperties = [
-            'TENANTS',
-            'LONG_TIME_FRAME_SCHEDULED_JOB_CRON',
```

Comment:
I see this variable is still being used in `.heroku.service.variables.enc`

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/226/commits/d6b5a41599d3e91fed30ce0b7d0e1daf00f8aea2